core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/WorkerBase.scala

```diff
@@ -74,19 +84,31 @@ abstract class WorkerBase(identifier: ActorVirtualIdentity) extends WorkflowActo
   def onModifyTuple(faultedTuple: FaultedTuple): Unit = {}
 
   def onStart(): Unit = {
-    logger.info(s"$identifier started!")
+    logger.logInfo(s"$identifier started!")
```

Comment:
please create a WorkflowLogger class.

Reply:
It's created at the WorkflowActor level.